README.md

```diff
@@ -412,6 +412,90 @@ kustomize build applications/pipeline/upstream/env/cert-manager/platform-agnosti
 ```
 This installs Argo with the runasnonroot emissary executor. Please note that you are still responsible for analyzing the security issues that arise when containers are run with root access and for deciding if the Kubeflow pipeline main containers are run as runasnonroot. It is generally strongly recommended that all user-accessible OCI containers run with Pod Security Standards [restricted](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted).
 
+#### Kubeflow Pipelines - Kubernetes Native API Mode
+
+Kubeflow Pipelines can be deployed in Kubernetes Native API mode, which stores pipeline definitions as Kubernetes Custom Resources instead of using external storage. This mode provides better integration with Kubernetes native tooling and GitOps workflows.
+
+#### Install Kubeflow Pipelines in Kubernetes Native API Mode
+
+```sh
+# For single-user deployment
+kustomize build applications/pipeline/upstream/env/cert-manager/platform-agnostic-k8s-native | kubectl apply -f -
+
+# For multi-user deployment
+kustomize build applications/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user-k8s-native | kubectl apply -f -

```

Comment:
Is there a reason that k8s-native is a separate option? why not incorporate it into the default for KFP ?

Reply:
k8s-native requires separate deployment option because it requires additional components  like cert-manager, admission webhooks to be deployed which are not used in the default deployment.

Reply:
but compared to platform-agnostic-multi-user which we already use. Why is it separate ? This is kubeflow platfrom we anyway use cert-manager etc.